crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -201,6 +201,22 @@ def compute_resources_available?
         true
       end
 
+      # Check for presence of HA setup, which is a requirement for non-disruptive upgrade
+      def ha_presence_check
```

Comment:
It's backport from master. And we want to return specific errors somehow, not just true/false